ravenframework/SupervisedLearning/ARMA.py

```diff
@@ -2527,6 +2528,7 @@ def setEngine(self,eng,seed=None,count=None):
       randomUtils.forwardSeed(count, engine=eng)
     self.randomEng = eng
 
+git
```

Comment:
extraneous entry?

Reply:
Yes, I saw that after I pushed. I thought I had pushed a fix earlier but maybe that's not showing up yet. I'll get that removed if it's still in the PR changes.